python/dgl/frame.py

```diff
@@ -792,3 +841,32 @@ def unpin_memory_(self):
         if necessary."""
         for column in self._columns.values():
             column.unpin_memory_()
+
+    def __astype_float(self, new_type):
```

Comment:
I would prefer having a single underscore.

Reply:
I've now changed it to one underscore, so that it's not a fully private function.  Thanks!